app/providers/repositories/conversation/index.js

```diff
@@ -10,12 +10,16 @@ class ConversationRepository extends BaseRepository {
     return await super.prepareParams(params)
   }
 
+  async findExistedPrivateConversationByCid(cid) {
```

Comment:
the naming says it's private conversation, but in reality it looks for ANY conversation.
Should we just remove this method and just use `findOne ` ?